### DISCUSSION TITLE
Unable to send any emails

### DISCUSSION THREAD

**Discussion by TheSunCat** [simple-login/app#1678] [Self-hosting question] [answered]
Hi! I've set up SimpleLogin on my homeserver, and I did not receive the confirmation email for signing up. Following the troubleshooting guide, I found that `swaks` shows the following error:
```
=== Trying mailserver.purelymail.com:25...
*** Error connecting to mailserver.purelymail.com:25:
*** 	IO::Socket::INET6: connect: timeout
```

My understanding is that this means my ISP is blocking port 25. Is there a way to tell postfix and SimpleLogin to use a different port, or bypass this somehow?
I'm new to mail stuff, so I'm not sure whether this is relevant, but Purelymail (my mail provider) allows one to add a custom domain and have it deliver to an "external server" or "relay" instead of Purelymail itself. Could this be used to somehow bypass the port 25 block?

**Comment by nguyenkims** [comment, marked as answer]
@TheSunCat unfortunately if your ISP blocks port 25, you can't connect to external email server to send emails. The solution is to rather run SimpleLogin on a VPS for example.

**Comment by TheSunCat** [comment]
Thanks for your reply! Is there really no way around this? I have used mail forwarding services such as Mailgun before, which allowed other self-hosted services to send emails.

**Reply by rehanone** [reply]
- It's not about you being able to send/forward emails that can use the relay services like mailgun. The issue here is other mail servers not being able to deliver email to your sl instance. If for instance I am using my gmail account to send you an email using an alias you provided, then at some point, gmail will find your hosted sl instance using DNS lookup and try to deliver my email to your server at port 25. However that will not work because your isp has blocked that port. There is no way around this as far as I can think of. So the only thing you can do is find a cloud hosted vm that doesn't block port 25. Many cloud providers do block port 25 as well. Like Aws doesn't allow port 25 I think. I have used linode in the past and by default they have port 25 blocked. But you can create a ticket to get it unblocked. Currently I am using ovh cloud and they don't have any restrictions on port 25 at this time. Hope this helps.

**Reply by TheSunCat** [reply]
Thanks for the explanation! I now understand why it won't work.
